misk/web/@misk/common/README.md

```diff
@@ -101,6 +101,24 @@ Contributing
 - Add window to javascript require include in `src/vendors.js`
 - Update `README.md` with a copy of the updated `package.json` list of packages
 
+#Adding a Cached Remote Asset
+- There are sometimes cases where remote assets should be cached in order to be included in the single line imports of `vendors.js` or `styles.js`
+- Add a new task to `package.json` under the metadata key   `"miskCachedUrls"` with the format as follows
+  ```JSON
+  "miskCachedUrls": {
+    "taskname": {
```

Comment:
taskname?

Reply:
maybe rename to myfont?

Reply:
This is the docs for how to add a static asset download task, taskname is just a filler.